daemon/config/README.md

```diff
@@ -82,21 +82,33 @@ Please note that your data history will be lost if you have modified `history` p
+|                  storage tiers                  |    `1`     | The number of storage tiers you want to have in your dbengine. Check the tiering mechanism in the [dbengine's reference](/database/engine/README.md#tiering). You can have up to 5 tiers of data (including the _Tier 0_). This number ranges between 1 and 5.                                                                                                                                                                                                                                                                                                                                                                      |
+|           dbengine page cache size MB           |    `32`    | Determines the amount of RAM in MiB that is dedicated to caching for _Tier 0_ Netdata metric values.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+|   dbengine tier **_`N`_** page cache size MB    |    `32`    | Determines the amount of RAM in MiB that is dedicated for caching Netdata metric values of the **_`N`_** tier. <br /> `N belongs to [1..4]`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         ||
+ |             dbengine disk space MB              |   `256`    | Determines the amount of disk space in MiB that is dedicated to storing _Tier 0_ Netdata metric values and all related metadata describing them. This option is available only for legacy configuration.                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
Suggest to ~strikethrough~  `dbengine disk space MB` and bold the "legacy configuration". Consider also replacing this generic statement with "for Netdata versions< 1.xyz"